tampermonkey/PPT_AutoComplete.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT Registration AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      15.8
+// @version      16.0
```

Comment:
Not sure how versioning is determined for the Tampermonkey scripts but this might be 15.9?